Fix poultry man menu

MrMisterMisterMister opened the pull request:
## Description

- Updated poultry man menu to the new menu that Finn made.
- Updated focusable3d in a cracky way.
- Updated index ordering for focusable items.
- Extreme crack. 
- I don't feel the labels. Might think of sth else. 
- Also added nest as healing station model

TODO
- Improve highlighting. (Maybe)

## Related Issue

#331 
#330 

## Checklist

_Put an `x` in the boxes that apply._

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I am merging into the correct branch.
- [ ] I have added appropriate documentation for the changes made.
- [x] I have followed the branch naming convention (`feature/hotfix/fix`), e.g., `feature/player_movement`.

## Screenshots (if appropriate):

PPS: 2
![image](https://github.com/user-attachments/assets/774376f4-7298-4183-83f7-733ed5a6075e)
PPS: 1
![image](https://github.com/user-attachments/assets/efd80a92-e841-4b48-9cc8-df61d26553e8)
PPS: 0
![Screenshot 2025-06-12 023349](https://github.com/user-attachments/assets/a21fb795-24c8-4008-adc5-e9e5c1496ded)

### Healing station
![Godot_v4 4 1-stable_win64_FdgqOiP8ON](https://github.com/user-attachments/assets/97a92a16-e0e3-412d-86d1-e7c51c5817a1)

Nrojt commented:
Is the audio effect no longer working with the flickering lights?

Nrojt commented:
Nitpick: why is the chicken cage not showing the player chicken but bob? Would make more sense to put the player chicken model there

Nrojt commented:
> > Nitpick: why is the chicken cage not showing the player chicken but bob? Would make more sense to put the player chicken model there
> 
> Because I like bog

I changed it

MrMisterMisterMister commented:
> > > Nitpick: why is the chicken cage not showing the player chicken but bob? Would make more sense to put the player chicken model there
> > 
> > 
> > Because I like bog
> 
> I changed it



> > > Nitpick: why is the chicken cage not showing the player chicken but bob? Would make more sense to put the player chicken model there
> > 
> > 
> > Because I like bog
> 
> I changed it



> > > Nitpick: why is the chicken cage not showing the player chicken but bob? Would make more sense to put the player chicken model there
> > 
> > 
> > Because I like bog
> 
> I changed it

This is democracy manifest
